### ISSUE TITLE
Changelog for new versions

### ISSUE THREAD

**Issue by crutchcorn** [oliverfindl/vue-svg-inline-plugin#10]
It would be nice to have a changelog of what's changing in each version.

Potentially even [autogenerated changelogs based off commit messages](https://unicorn-utterances.com/posts/setup-standard-version/)?

**Comment by oliverfindl** [comment]
Hello,

thanks for idea, but unfortunately this package is kinda dead and deserted. Personally, I failed to start using it in my projects and therefore I have no motivation to contribute to it anymore. I created this package to solve some issues of [vue-svg-inline-loader](https://github.com/oliverfindl/vue-svg-inline-loader), but failed at that task, because of [Vue](https://github.com/vuejs/vue-next) limitations. I would recommend to check [vue-svg-inline-loader](https://github.com/oliverfindl/vue-svg-inline-loader) or [vue-svg-inline-component](https://github.com/oliverfindl/vue-svg-inline-component) instead.

Thanks.
